File: app/src/main/java/com/example/achalker/starbuzz/FoodCategoryActivity.java
package com.example.achalker.starbuzz;

import android.app.Activity;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.ArrayAdapter;
import android.widget.ListView;

public class FoodCategoryActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_food_category);

        ArrayAdapter<Food> listAdapter = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, Food.foods);
        ListView listFoods = (ListView) findViewById(R.id.list_foods);
        listFoods.setAdapter(listAdapter);
    }
}

File: app/src/main/java/com/example/achalker/starbuzz/FoodCategoryActivity.kt
package com.example.achalker.starbuzz

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import android.view.View
import android.widget.ArrayAdapter
import android.widget.ListView
import com.example.achalker.starbuzz.model.Food

class FoodCategoryActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_food_category)
        val listAdapter = ArrayAdapter(this, R.layout.list_item, Food.foods.map { it.name })
        val listFoods = findViewById<View>(R.id.list_foods) as ListView
        listFoods.adapter = listAdapter
    }
}